LayoutTests/accessibility/mac/html-section-elements.html

```diff
@@ -41,11 +46,35 @@
           shouldBe("body.childAtIndex(5).subrole", "'AXSubrole: AXLandmarkComplementary'");
 
           // When a header and footer are inside an article, they should not be landmarks.
+          shouldBe("body.childAtIndex(6).domIdentifier", "'article'");
+          shouldBe("body.childAtIndex(6).childAtIndex(0).domIdentifier", "'article_header'");
+          shouldBe("body.childAtIndex(6).childAtIndex(0).role", "'AXRole: AXGroup'");
           shouldBe("body.childAtIndex(6).childAtIndex(0).subrole", "'AXSubrole: '");
+          shouldBe("body.childAtIndex(6).childAtIndex(1).domIdentifier", "'article_footer'");
+          shouldBe("body.childAtIndex(6).childAtIndex(1).role", "'AXRole: AXGroup'");
+          shouldBe("body.childAtIndex(6).childAtIndex(1).subrole", "'AXSubrole: AXFooter'");
+
+	  // FIXME: How can we access <section>?
```

Comment:
The index value of childAtIndex(n) seems unusual. It counts all the children of the body element, but the <section> element doesn't have an index. How can we access the <section> element using the childAtIndex method?

Reply:
So this is actually using the `AccessibilityUIElement::childAtIndex` API (because `body` was retrieved via `accessibilityController.focusedElement`). This means it's using the accessibility tree children, not the DOM children, and the accessibility tree can differ quite a lot from the DOM.

What's very likely happening here is that the section element is determined to be "ignored" by accessibility (you can search for that term in the codebase to get a lot of context), which effectively means it's not part of the AX tree, and thus cannot be retrieved via `childAtIndex`. This is fine, I don't think we need to test section here (unless there's something I'm missing), so we can remove the FIXME.